s2i/pom.xml

```diff
@@ -80,21 +80,36 @@
         <version>${maven.exec.plugin.version}</version>
         <executions>
           <execution>
-            <id>exec</id>
+            <id>syndesis-builder-image-generator</id>
             <phase>package</phase>
             <goals>
               <goal>exec</goal>
             </goals>
+            <configuration>
+              <executable>java</executable>
+              <arguments>
+                <argument>-jar</argument>
+                <argument>${project.build.directory}/syndesis-builder-image-generator.jar</argument>
+                <argument>--to=${basedir}/src/main/docker/project</argument>
+              </arguments>
+            </configuration>
+          </execution>
+          <execution>
+            <id>copy-local-repo</id>
+            <phase>package</phase>
+            <goals>
+              <goal>exec</goal>
+            </goals>
+            <configuration>
+              <executable>bash</executable>
```

Comment:
won't work on windows obviously. Maybe we should add this to the README.

Reply:
This would be just one item in a long list no?